typescript/sdk/src/deploy/middleware/LiquidityLayerApp.ts

```diff
@@ -100,8 +141,10 @@ export class LiquidityLayerApp<
       .message;
     const nonce = matchingLogs.find((_) => _!.name === 'BridgedToken')!.args
       .nonce;
-    const remoteChain =
-      message.chain === Chains.fuji ? Chains.goerli : Chains.fuji;
+    const remoteChain = chain === Chains.fuji ? Chains.goerli : Chains.fuji;
```

Comment:
this looks fragile

Reply:
Agreed, filed https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/1327